apps/web/app/(app)/[emailAccountId]/assistant/RuleForm.tsx

```diff
@@ -857,7 +859,7 @@ export function RuleForm({
               variant="secondary"
               size="sm"
               onClick={() => {
-                append({ type: ActionType.LABEL });
+                append({ type: ActionType.LABEL, delayInMinutes: null });
```

Comment:
remove delayinminutes: null. just keep it emoty like all the other fields we could have here but are not set